src/frontend/components/Row/Row.jsx

```diff
@@ -16,10 +20,10 @@ export const Row = ({ cat, title }) => {
   const Games = videos.filter((vid) => vid.categoryName === cat);
   const ArrowHandler = (dir) => {
     if (dir === "right") {
-      setVal(v => v - 20);
+      setVal((v) => v - 20);
```

Comment:
try to give proper variable names!
eg. `prevValue` or `prevState`

Reply:
Again, rightly pointed out by @vijaytembugade!